package.json

```diff
@@ -8,27 +8,28 @@
     "apollo-link-http": "^1.3.3",
     "d3-scale": "^2.0.0",
     "eslint-plugin-react": "^7.7.0",
+    "extract-react-intl-messages": "^0.7.1",
     "graphql": "^0.13.0",
     "graphql-tag": "^2.7.3",
     "graphql-tools": "^2.20.2",
+    "husky": "^0.14.3",
+    "lint-staged": "^4.3.0",
     "lodash-es": "^4.17.5",
     "numeral": "^2.0.6",
     "polished": "^1.8.1",
+    "prettier": "^1.7.4",
     "react": "^16.2.0",
     "react-apollo": "^2.1.0",
+    "react-app-rewire-styled-components": "^3.0.0",
+    "react-app-rewired": "^1.3.4",
     "react-dom": "^16.2.0",
+    "react-helmet": "^5.2.0",
     "react-intl": "^2.4.0",
     "react-router-dom": "^4.2.2",
     "react-scripts": "^1.1.1",
     "styled-components": "^2.2.3",
     "text-gradient": "^0.2.0",
-    "typography": "^0.16.6",
-    "extract-react-intl-messages": "^0.7.1",
-    "husky": "^0.14.3",
-    "lint-staged": "^4.3.0",
-    "prettier": "^1.7.4",
-    "react-app-rewire-styled-components": "^3.0.0",
```

Comment:
This package caused lots of issues earlier, are you sure you want to remove this?

Reply:
He hasn’t removed these. Looks like he’s just alphabetisized the file 😊